meshtastic/module_config.proto

```diff
@@ -201,6 +201,15 @@ message ModuleConfig {
      * Only applicable if the board uses pull-up resistors on the pin
      */
     bool use_pullup = 8;
+
+    /* If this is set, message are send to channel set in "sendTo" else a "NodeID"*/
+    bool sendAsDM = 9;
```

Comment:
Sorry for nitpicking, but all other settings here use snake_case, so I think `send_as_dm` and `send_to` would be better. Also, the comment needs to be updated here.

Reply:
Something like 
```
 /* If this is set true, messages are send to UserID.
  * Set to false, messages are send channel index.
  * See "sendTo".
  */
    bool send_as_dm = 9;

Reply:
I would say: 

```
 /* If this is set true, messages are send as a direct message to the NodeNum set in "sendTo". 
  * If set to false, messages are broadcasted to a channel with the index set in "sendTo".
  */
```

Reply:
Done.